projects/foreign_testing/orchestration/foreign_testing.py

```
@@ -49,12 +49,26 @@ def prepare():
+    Submit a FOURNOS deployment job
+
+    Args:
+        project_path: Path to the project source directory (will be passed as --project-source)
+    """
+    # Build the command arguments
+    args = ["deploy"]
+
+    if project_path:
+        args.extend(["--project-source", str(project_path)])
+
+    logger.info(f"Submitting deployment with args: {args} and extra_args: {extra_args}")
```

Ruff (0.15.10)

[error] Undefined name `extra_args`

(F821)